pallets/msa/src/benchmarking.rs

```diff
@@ -157,16 +157,21 @@ benchmarks! {
 	}: _(RawOrigin::Signed(caller_and_delegator_public_key), new_public_key)
 
 	retire_msa {
-
+		let s in 5 .. MAX_NUMBER_OF_PROVIDERS_PER_DELEGATOR;
 		let caller: T::AccountId = whitelisted_caller();
-
-		// Create a MSA account
-		assert_ok!(Msa::<T>::create(RawOrigin::Signed(caller.clone()).into()));
```

Comment:
don't you need to create an MSA first to be able to retire it?

Reply:
I think we were taking a shortcut here by using `Delegator(1)` (MSA id 1) in the `add_provider` call.  I could create it with `create_account_with_msa_id()` if you think that is better.